cmd/zetacored/root.go

```diff
@@ -212,7 +292,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig types.EncodingConfig) {
 		queryCommand(),
 		txCommand(),
 		docsCommand(),
-		ethermintclient.KeyCommands(app.DefaultNodeHome),
+		cosmosevmcmd.KeyCommands(app.DefaultNodeHome, true), // TODO evm: second arg default to eth keys?
```

Comment:
I'm not sure honestly I would keep the legacy behavior for this one, we generally use Cosmos address format for Cosmos command